cmd/kogito/command/deploy/delete_service.go

```diff
@@ -48,8 +45,8 @@ func (i *deleteServiceCommand) RegisterHook() {
 	i.command = &cobra.Command{
 		Example: "delete-service example-drools --project kogito",
 		Use:     "delete-service NAME [flags]",
-		Short:   "Deletes a Kogito service deployed in the namespace/project",
-		Long:    `delete-service will exclude every OpenShift/Kubernetes resource created to deploy the Kogito service into the namespace.`,
+		Short:   "Deletes a Kogito build/service deployed in the namespace/project",
```

Comment:
The build is part of the service, we can keep the "service" only. We might add a `kogito delete build` if necessary in the future.

Reply:
done